From 729fc6f0139f63329b544a15cc34ec8e13ed385e Mon Sep 17 00:00:00 2001
From: Lucas Stach <l.stach@pengutronix.de>
Date: Tue, 23 Jun 2015 11:51:50 +0200
Subject: [PATCH 087/139] staging: etnaviv: don't override platform provided
 IRQ flags

The platform should specify the appropriate IRQ flags and it's
a really bad idea to override them in individual drivers.

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 drivers/staging/etnaviv/etnaviv_gpu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/etnaviv/etnaviv_gpu.c b/drivers/staging/etnaviv/etnaviv_gpu.c
index da25793..8fa3b00 100644
--- a/drivers/staging/etnaviv/etnaviv_gpu.c
+++ b/drivers/staging/etnaviv/etnaviv_gpu.c
@@ -1189,8 +1189,8 @@ static int etnaviv_gpu_platform_probe(struct platform_device *pdev)
 		goto fail;
 	}
 
-	err = devm_request_irq(&pdev->dev, gpu->irq, irq_handler,
-			IRQF_TRIGGER_HIGH, dev_name(gpu->dev), gpu);
+	err = devm_request_irq(&pdev->dev, gpu->irq, irq_handler, 0,
+			       dev_name(gpu->dev), gpu);
 	if (err) {
 		dev_err(dev, "failed to request IRQ%u: %d\n", gpu->irq, err);
 		goto fail;
-- 
2.5.1

